src/containerapp/azext_containerapp/_clients.py

```diff
@@ -1716,3 +1716,254 @@ def list_by_resource_group(cls, cmd, resource_group_name, formatter=lambda x: x)
                 env_list.append(formatted)
 
         return env_list
+
+    @classmethod
+    def show_certificate(cls, cmd, resource_group_name, name, certificate_name):
```

Comment:
Could we separate  to CertifaicteClient?

Reply:
It is consisting with ManagedEnvironmentClient.